src/main/java/seedu/bigpp/ui/UI.java

```diff
@@ -34,7 +34,11 @@ public static void updateUI() {
 
         switch (uiState) {
         case MAIN_MENU:
-            printMainMenu();
+            if (isStartingMainMenu()) {
+                printStartingMainMenu();
+            } else {
+                printMainMenu();
+            }
             break;
```

Comment:
if starting mainmenu, no back feature. if mainmenu visited after other menu, back feature will be seen and can be used